controllers/containerimagebuild_controller.go

```diff
@@ -49,13 +52,13 @@ func (r *ContainerImageBuildReconciler) Reconcile(req ctrl.Request) (ctrl.Result
 	}
 
 	// ignore resources that have been processed on start
-	if len(build.Status.State) != 0 {
+	if build.Status.State != "" {
```

Comment:
Not familiar enough with the codebase to say this suggestion definitely makes sense, but wonder if it'd be better if these had an initial state ("Uninitialized" or something) so we can compare it to something concrete.

Reply:
Ideally, I'd like to leverage kubebuilder comment annotations to assign a default value (then we could do something like that) but I haven't been able to get it to work. I'm definitely going to spend some more time on this once the _first flush_ is out.